jdaviz/configs/default/plugins/export_plot/export_plot.vue

```diff
@@ -25,11 +26,110 @@
         <v-btn
          color="primary"
          @click="() => save_figure('svg')"
+         :disabled="movie_recording"
         >
          Export to SVG
         </v-btn>
        </v-list-item>
       </v-list>
+
+      <v-row v-if="config==='cubeviz'">

```

Comment:
```suggestion
     <v-row v-if="config==='cubeviz' && viewer_selected!='spectrum-viewer'">
```

This would hide the `Export to Video` expansion tray completely if `spectrum-viewer` is selected - I kind of like that better than just graying-out the export button in that case, I could see it being a little confusing for the user that all the options are there but they can't actually hit the button. I don't feel strongly about this though and would be ok merging with the current behavior.